File: java/src/main/java/itp/itp1_2/ITP1_2_B.java
package itp.itp1_2;

import java.util.Scanner;

public class ITP1_2_B {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    
    int a = sc.nextInt();
    int b = sc.nextInt();
    int c = sc.nextInt();

    System.out.println(a < b && b < c ? "Yes" : "No");

    sc.close();
  }
}

File: kotlin/src/itp/itp1_2/ITP1_2_B.kt
package itp.itp1_2

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val a = sc.nextInt()
    val b = sc.nextInt()
    val c = sc.nextInt()
    println(if (b in (a + 1) until c) "Yes" else "No")
}